marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/jobInfo/jobInfo.sjs

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2012-2019 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+'use strict';
+
+var model;
+
+let res = {
+  "modelName": model,
```

Comment:
I would make the following changes:

- modelName = entityCollection
- jobId = latestJobId
- jobTime = latestJobDateTime

I like to use "DateTime" as the suffix to imply that a value is an xs.dateTime, and likewise for "Date" (xs.date) and "Time" (xs.time).

Reply:
Included in https://github.com/marklogic/marklogic-data-hub/pull/3011/commits/e1d00a2bc5e0cc4d4afe9e00cfd0fc146f4f53aa